charts/k8ssandra/templates/cassandra/cassdc.yaml

```diff
@@ -1,5 +1,8 @@
 {{- $datacenter := (index .Values.cassandra.datacenters 0) -}}
-{{- $medusaImage := (printf "%s:%s" .Values.backupRestore.medusa.image.repository .Values.backupRestore.medusa.image.tag) -}}
+{{- $medusaImage := (printf "%s:%s" .Values.medusa.image.repository .Values.medusa.image.tag) -}}
+{{- $cassandrUid := 999 -}}
```

Comment:
Typo? Can't seem to find a match for this (missing one a)

Reply:
Yeah, and I will remove these. They are no longer used since I took out the SecurityContext.